app/assets/javascripts/incidents.subquestions.js

```diff
@@ -75,4 +78,19 @@ $(function(){
     $('#multiple-agencies-info-text').toggle($('#screener_multiple_agencies_true').is(':checked'));
     $('#shots-fired-info-text').toggle($('#screener_shots_fired_true').is(':checked'));
   }).change();
+
+
+  function medicalAid() {
```

Comment:
Yeah, agreed with @everett-wetchler on this and then next comment.

This can be concisely written as something like:
```js
$('.medical-aid-received input[type="radio"]').on('click', function () {
  if ($('input[value="No medical assistance or refused assistance"]').is(':checked')) {
    disableQuestions('.additional-question.pre-existing-condition');
  } else {
    enableQuestions('.additional-question.pre-existing-condition');
  }
});
```